metadata-ingestion/src/datahub/ingestion/source/identity/okta.py

```diff
@@ -8,9 +8,6 @@
 from typing import Dict, Iterable, List, Optional, Set, Union
 
 import nest_asyncio
-from okta.client import Client as OktaClient
```

Comment:
pylint insisted upon this in order to let me commit

Reply:
good with me, as long as `ruff` is happy with it